modules/criteoBidAdapter.js

```diff
@@ -391,7 +391,7 @@ export const spec = {
     const interpretedResponse = CONVERTER.fromORTB({response: response.body, request: request.data});
     const bids = interpretedResponse.bids || [];
 
-    const fledgeAuctionConfigs = deepAccess(response.body, 'ext.igi')?.filter(igi => isArray(igi?.igs))
+    const fledgeAuctionConfigs = response.body?.ext?.igi?.filter(igi => isArray(igi?.igs))
```

Comment:
For some reason I thought this would require optional chaning on the function call as well (`filter?.(...)`) but it does appear to work. Do you know if it's guaranteed to work? I can't find it in the standard and I'm worried it might just be some browsers' optimization.

Reply:
on second thought, the previous version was not chaining the call either, so it's a moot point.

Reply:
I believe this will throw `TypeError` if `response.body.ext.igi` exists but does not have a `filter` method. However, in its current form it is functionally equivalent to the previous version.